scripts/cms-any/merge-ups.php

```diff
@@ -51,7 +55,7 @@
       actions: write
     steps:
       - name: Merge-up
-        uses: silverstripe/gha-merge-up@v1
+        uses: silverstripe/gha-merge-up@v$version
```

Comment:
Seems more sensible than having two copies of exactly the same script with only this difference